python/foglamp/tasks/north/sending_process.py

```diff
@@ -818,6 +818,12 @@ def _fetch_configuration(self, cat_name=None, cat_desc=None, cat_config=None, ca
             self._core_microservice_management_client.create_configuration_category(config_payload)
             _config_from_manager = self._core_microservice_management_client.get_configuration_category(category_name=cat_name)
 
+            # Check and warn if pipeline exists in North service
+            if 'filter' in _config_from_manager:
+                _LOGGER.warning('Filter pipeline is not supported on Python North service [%s], plugin [%s]',
```

Comment:
North is not a service, but a `task` (or say `task instance`) 


FYI, for south the message is 

`_LOGGER.warning('South Service [%s] does not support the use of a filter pipeline.', self._name)`